README.md

```diff
@@ -38,7 +40,7 @@ added: REPLACEME
   * `positionals` {string\[]}, containing positional arguments.
 
 Provides a higher level API for command-line argument parsing than interacting
-with `process.argv` directly.
+with `process.argv` directly. Takes a specification for the expected options and returns a structured object corresponding to passed arguments.
```

Comment:
Trying a variation to cover options and positionals. I wanted to try and spin the story the config is for everything you expect.

This does give positional more weight than has in the config, but values/positionals are more equally righted in  the result.

```suggestion
with `process.argv` directly. Takes a specification for the expected arguments and returns a structured object with the parsed options and positionals.
```

Reply:
SGTM